common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -112,6 +112,33 @@ def test_add_response_button_closed_thread(self):
 
 
 @attr('shard_1')
+class DiscussionHomePageTest(UniqueCourseTest):
+    """
+    Tests for the discussion home page.
+    """
+
+    SEARCHED_USERNAME = "gizmo"
+
+    def setUp(self):
+        super(DiscussionHomePageTest, self).setUp()
+        CourseFixture(**self.course_info).install()
+        AutoAuthPage(self.browser, course_id=self.course_id).visit()
+        self.page = DiscussionTabHomePage(self.browser, self.course_id)
+        self.page.visit()
+
+    def test_new_post_button(self):
+        """
+        Scenario: I can create new posts from the Discussion home page.
+            Given that I am on the Discussion home page
+            When I click on the 'New Post' button
+            Then I should be shown the new post form
+        """
```

Comment:
Note: I tried to actually fill in the form and submit it, but that doesn't work with the mock comment service.